portfolio/src/main/webapp/Projects/snakeProject/snake.js

```diff
@@ -1,33 +1,40 @@
 function Snake() {
-    this.x = 300;
-    this.y = 300;
-    this.xspeed =0;
-    this.yspeed =1;
-    this.total = 0;
-    this.tail = [];
-
-  this.show = function(){
+
```

Comment:
nit: Personally I would remove any leading blank lines inside a function like these but up to you. If it was added by the formatter then no worries.